PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+## Description
+
+Please include a summary of the change.
+
+<!--- 
+Optional if an issue is fixed:
+Fixes #(issue)
+-->
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist:
+
+- [ ] I have selected the correct base branch.
+- [ ] I have fixed the tests and extended if necessary.
+- [ ] I have performed a self-review of my own code.
```

Comment:
I also should test my code on the live network. 

And we should add, i ran `npm run build` and tested the resulting file from the `dist` folder in a browser.